pkg/minikube/cluster/cluster_darwin.go

```diff
@@ -49,7 +49,9 @@ type xhyveDriver struct {
 	Memory         int
 	PrivateKeyPath string
 	UUID           string
-	NFSShare       bool
+	NFSShareEnable bool
```

Comment:
If PR https://github.com/zchee/docker-machine-driver-xhyve/pull/193 is merged, `NFSShareEnable` will become obselete and we'll be able to remove it.